### PR TITLE
escape column names in sql statements #2141

### PR DESCRIPTION
**Pull request by prondubuisi**
delete orphan functions
first iteration

### CODE REVIEW

File: common/logging/EventAuditLogger.php [hunk 1]
```diff
@@ -315,7 +315,7 @@ public function getEvents($params)
         } else {
             // do the query
             $sqlBindArray = array();
-            $sql = "SELECT $cols FROM log WHERE date >= ? AND date <= ?";
+            $sql = "SELECT " . escape_sql_column_name(process_cols_escape($cols), array('log')) . " FROM log WHERE date >= ? AND date <= ?";
```
Comment (bradymiller) [inline common/logging/EventAuditLogger.php]: Check out the default $cols on line 218. Note the DISTINCT keyword will break our function. Recommend reverting change above and noting this as an exception. We can work on way to make this work in the future.

File: library/auth.inc [hunk 2]
```diff
@@ -206,27 +206,13 @@ function authCheckExpired($user)
...
-    }
-
-    for ($iter = 0; $row = sqlFetchArray($rez); $iter++) {
-        $tbl[$iter] = $row;
-    }
-
-    return $tbl;
-}
 
 function getProviderList($cols = '*', $limit = 'all', $start = '0')
```
Comment (bradymiller) [inline library/auth.inc]: We can remove this function. This one is not used. There is another one with same name in the care coordination module that is used.

File: library/forms.inc [hunk 3]
```diff
@@ -1,16 +1,10 @@
...
 $GLOBALS['form_exit_url'] = "javascript:parent.closeTab(window.name, false)";
 
-function getFormById($id, $cols = "*")
-{
-    $sql = "SELECT $cols FROM forms WHERE id = ? AND deleted = 0 ORDER BY date DESC limit 0,1";
-    //echo $sql . "<br />";
-    return sqlQuery($sql, array($id));
-}
 
 function getFormInfoById($id, $cols = "*")
```
Comment (bradymiller) [inline library/forms.inc]: we can remove this function. the only use of this function is actually commented out

**Comment by bradymiller** [comment]
hi @prondubuisi ,
Review complete. This will be ready for codebase when issues addressed. thanks! -brady

**Comment by prondubuisi** [comment]
@bradymiller I have pushed the changes

**Comment by bradymiller** [comment]
hi @prondubuisi ,
Code looks good and bringing into codebase. thanks for the code improvements! -brady

**Pull request merged**